### PR TITLE
Mandelbrot release dynamic comparison

### PR DESCRIPTION
**Pull request by Ldelaney**
Change the graph that compares the dynamic domain iterator against the dynamic range iterator to use the release version of mandelbrot instead of mandelbrot-unrolled. Adds the relevant files necessary to do so, including the modified version of the release version.

### CODE REVIEW

File: test/studies/shootout/mandelbrot/ldelaney/mandelbrot-release-dynamic.chpl [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+type eltType = uint(bitsPerElt);   // element type used to store the image
+
+
+proc main() {
+  const ydim = 0..#n,                          // the image's y dimension
+        xdim = 0..#divceilpos(n, bitsPerElt);  // the compacted x dimension
+
+  var image : [ydim, xdim] eltType;            // the compacted bitmap image
+
+    forall (y, xelt) in dynamic({ydim, xdim}, chunkSize) {
```
Comment (bradcray) [inline test/studies/shootout/mandelbrot/ldelaney/mandelbrot-release-dynamic.chpl]: In the above, I think we should move away from declaring xdim and ydim, simply declare image as:

    var image: [0..#n, 0..#divceilpos(...)];

and then make the forall w/ dynamic be over image.domain

If the high bound for ydim seems too complex to jam into an array declaration, maybe we could just symbolically name its upper bound?

**Comment by benharsh** [comment]
Other than Brad's comment, I think you still need to list this in test/GRAPHFILES.

**Comment by Ldelaney** [comment]
It should already be listed in test/GRAPHFILES. I am changing the graph I made in my original pull request to use the release version and a modified version of the release version (instead of mandelbrot-unrolled). 

test/studies/shootout/mandelbrot/mandelbrot-dynamicIter.graph is not a new graph, and it has been modified instead of being replaced.

**Comment by benharsh** [comment]
Oops, you're right, nevermind!

File: test/studies/shootout/mandelbrot/ldelaney/mandelbrot-release-dynamic.chpl [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+             chunkSize = 1;        // dynamic iterator's chunk size
+
+param bitsPerElt = 8;              // # of bits to store per array element
+type eltType = uint(bitsPerElt);   // element type used to store the image
+
+
+proc main() {
+  const xdim = 0..#divceilpos(n, bitsPerElt);  // the compacted x dimension
+
+  var image : [0..#n, xdim] eltType;            // the compacted bitmap image
```
Comment (bradcray) [inline test/studies/shootout/mandelbrot/ldelaney/mandelbrot-release-dynamic.chpl]: @Ldelaney and @benharsh : When moving this to the release, what do you think about simply capturing the expression divceilpos(n, bitsPerElt) in a const named something like 'nPacked', eliminating xdim, and using an anonymous range for the x dimension as well?  I.e.,

    const nPacked = divceilpos(n, bitsPerElt);

    var image: [0..#n, 0..#nPacked] eltType;

This has the advantage of not having an xdim without a ydim, avoids the ongoing strangeness of why the second dimension is 'x' (though I guess the loop below preserves it in any case), and makes the dimensions of image seem slightly more symmetrical in how they're declared.

Comment (benharsh) [inline test/studies/shootout/mandelbrot/ldelaney/mandelbrot-release-dynamic.chpl]: That sounds like a good idea to me!

**Pull request merged**